flight/Modules/System/systemmod.c

```diff
@@ -280,6 +282,14 @@ static void systemPeriodicCb(UAVObjEvent *ev, void *ctx, void *obj_data, int len
 	// Update the modem status, if present
 	updateRfm22bStats();
 
+#ifndef NO_SENSORS
+	if (config_check_needed) {
+		configuration_check();
+	}
+
+	config_check_needed = false;
```

Comment:
Thanks-- I'm used to doing it outside but without synchronization it's really important to get it in.